webapp/pages/post/_id/_slug/index.vue

```diff
@@ -216,6 +221,12 @@ export default {
           id: this.$route.params.id,
         }
       },
+      update({ Post }) {
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-01-20T16:36:54Z" title="Monday, January 20th 2020, 5:36:54 pm +01:00">Jan 20, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -215,6 +223,26 @@ export default {
           id: this.$route.params.id,
         }
       },
+      update({ Post }) {
```


I like this refactoring :+1: